frontend/integration-tests/views/crud.view.ts

```diff
@@ -169,10 +176,12 @@ export const deleteResource = async (resource: string, kind: string, name: strin
 // then navigates back to the original url.
 export const createNamespacedTestResource = async (kindModel, name) => {
   const next = await browser.getCurrentUrl();
-  await browser.get(`${appHost}/k8s/ns/${testName}/${kindModel.plural}/~new`);
+  await browser.get(`${appHost}/k8s/ns/${testName}/${kindModel.plural}`);
+  await isLoaded();
+  await clickCreateWithYAML();
   await yamlView.isLoaded();
   const content = await yamlView.getEditorContent();
-  const newContent = _.defaultsDeep(
+  const newContent = await _.defaultsDeep(
```

Comment:
No reason for `await` here.

Reply:
Sure, dropping it. It must be the flakes that fooled me that fixed it..